src/components/ItemEditorPage/TopPanel/TopPanel.tsx

```diff
@@ -26,7 +27,7 @@ export default class TopPanel extends React.PureComponent<Props, State> {
 
   setShowRejectionModal = (showRejectionModal: RejectionType | null) => this.setState({ showRejectionModal })
 
-  renderPage = (collection: Collection, curation: CollectionCuration | null, itemsCuration: ItemCuration[]) => {
+  renderPage = (collection: Collection, collectionItems: Item[], curation: CollectionCuration | null, itemsCuration: ItemCuration[]) => {
```

Comment:
what do you think about keeping both this method and renderButton with the same argument order?

```ts
renderPage = (collection: Collection, collectionItems: Item[], curation: CollectionCuration | null, itemsCuration: ItemCuration[]) => {
renderButton = (type: ButtonType, collection: Collection, curation: CollectionCuration | null, collectionItems?: Item[]) => {
```

I understand that collectionItems must be at the end to be partial but maybe?

```ts
renderPage = ({
  collection: Collection, collectionItems: Item[], curation: CollectionCuration | null, itemsCuration: ItemCuration[]
}) => {
renderButton = (type: ButtonType, {collection: Collection, curation: CollectionCuration | null, collectionItems?: Item[]}) => {
```

same goes for `renderTPButtons`